scss/forms/_form-range.scss

```diff
@@ -12,18 +12,12 @@
   appearance: none;
 
   &:focus {
-    // Boosted mod: better focus visibility
+    outline: 0;
```

Comment:
The Boosted mod is reverted because it is fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1437? Why don't we let it until https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1437 is merged?

Reply:
It's the same as Bootstrap 🤷

Reply:
Yes but it wasn't before in Boosted. When you focus the input range with the keyboard, in this PR, we don't have anymore the black border around the input range. It is a regression.

Reply:
Shall we put this black border back or is it enough to have the range thumb turn orange when focused?

Reply:
Let's keep what was done before:
```scss
  &:focus {
    // Boosted mod: better focus visibility
    // Pseudo-elements must be split across multiple rulesets to have an effect.
    // No box-shadow() mixin for focus accessibility.
    &::-webkit-slider-thumb {
      border-color: $form-range-thumb-active-border-color;
      box-shadow: $form-range-thumb-focus-box-shadow;
    }
    &::-moz-range-thumb {
      border-color: $form-range-thumb-active-border-color;
      box-shadow: $form-range-thumb-focus-box-shadow;
    }
    // End mod
  }
```